test/lit/passes/optimize-instructions-mvp.wast

```diff
@@ -11575,10 +11595,22 @@
       (local.get $x)
       (i32.const 0)
     ))
+    (drop (i32.ge_u
+      (i32.load
+       (i32.const 0)
+      )
```

Comment:
Same comment as the other PR, please use `(local.get $x)` (1 line) rather than a load with a constant (3 lines).

Reply:
(this was my confusion in the other PR)